src/schema/v2/order/types/DisplayTexts.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import type { ResolverContext } from "types/graphql"
+import { OrderJSON } from "./exchangeJson"
+import { formatMarkdownValue } from "../../fields/markdown"
+import { FormatEnums } from "../../input_fields/format"
+
+const DisplayTextsType = new GraphQLObjectType<any, ResolverContext>({
+  name: "DisplayTexts",
+  description:
+    "Display texts for the order based on its state and order shipping/payment states",
+  fields: {
+    titleText: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Title text to display for the order",
+    },
+    messageText: {
+      type: GraphQLString,
+      description: "First paragraph of the message text for the order",
+      args: {
+        format: {
+          type: FormatEnums,
+          defaultValue: "markdown",
+        },
+      },
+      resolve: ({ messageText }, { format }) => {
+        if (!messageText) return null
+        return formatMarkdownValue(messageText, format)
+      },
+    },
+  },
+})
+
+export const DisplayTexts: GraphQLFieldConfig<OrderJSON, ResolverContext> = {
+  description:
+    "Display texts for the order based on its buyer_state and order shipping/payment states",
+  type: new GraphQLNonNull(DisplayTextsType),
+  resolve: (order) => resolveDisplayTexts(order),
+}
+
+const formatMessage = (parts: string[]) => parts.join("<br/><br/>")
```

Comment:
Looks like this helper is not necessary?

Reply:
@starsirius  - It is used as my idea is to make message as an array or parts and those parts are joined by beak line. For now it's only one line there so it's a bit redundant - but I'll add to the message in a follow up pr. That is where we need to use shipping details so was waiting for the other exchange PR.